.env-example

```diff
@@ -0,0 +1,6 @@
+DATABASE_URL=
+DEV_DATABASE_URL=postgres://<db_user>:<db_password>@127.0.0.1:5432/dev_db
+TEST_DATABASE_URL=postgres://<db_user>:<db_password>@127.0.0.1:5432/test_db
+
+db_user: name of the owner of the database. Moslty always 'postgres'
+db_password: passwoed set for the database
```

Comment:
hi @uthdev , I see you connecting to the database using the URL, could there be any need for the `db_user` and `db_password` keys, Also they're not assigned to any specific information

Reply:
@Oliver-ke this is just a guide on how to make the connection string. I put those there as additional information for anyone setting up to know what the db_user and db_password are. Not that they should create two new keys db_user and db_password after creating the connection string.